Jenkinsfile.deploy

```diff
@@ -27,6 +27,7 @@ pipeline {
         // Build&Test information
         booleanParam(name: 'SMOKE_TESTS_ONLY', defaultValue: false, description: 'If only smoke tests should be run. Default is full testing.')
         booleanParam(name: 'SKIP_TESTS', defaultValue: false, description: 'Skip tests')
+        string(name: 'BDD_TEST_TAGS', defaultValue: '', description: 'Execute only a subset of BDD tests')
```

Comment:
This should not be committed here, It will be handled in https://issues.redhat.com/browse/KOGITO-4307